coderd/workspaces.go

```diff
@@ -950,6 +951,28 @@ func (api *API) putWorkspaceDormant(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// We don't need to notify the owner if they are the one making the request.
+	if req.Dormant && apiKey.UserID != workspace.OwnerID {
+		initiator, err := api.Database.GetUserByID(ctx, apiKey.UserID)
+		if err != nil {
+			api.Logger.Warn(ctx, "failed to fetch the user that marked the workspace as dormant", slog.Error(err))
```

Comment:
Let's log the workspace name as well, otherwise this warning isn't very helpful.